ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/HardDisks.py

```diff
@@ -75,9 +75,17 @@ def process(self, device, results, log):
                 size = int(drive.Size)
             except TypeError:
                 size = 0
-            serialNumber = drive.SerialNumber.strip() if hasattr(drive, 'SerialNumber') else ''
+
+            # drive.SerialNumber could be None.  let's make it ''
+            serialNumber = ''
+            if hasattr(drive, 'SerialNumber'):
+                if drive.SerialNumber is None:
```

Comment:
i change drive.SerialNumber to '' if it comes in as None so that it will work correctly in make_disk_ids